### PR TITLE
document use of zat to deploy apps. Fixes #21

### LINKED ISSUE #21: Documentation for packaging
The README covers everything from installation to running locally and testing. The area that is missing, is packaging and deployment.

I assume the following is still relevant for this repo:

``` bash
cd dist
zat validate
zat package
```

And, then manually uploading the .zip file from the `/tmp` to your Zendesk instance.

Testing by adding `?zat=true` to the URL.

### PR DESCRIPTION
**Pull request by sandlerr**
cc @zendesk/vegemite

### CODE REVIEW

**Comment by danielbreves** [comment]
Hah I was about to work on this.

**Comment by sandlerr** [comment]
sorry, forgot that you are own the docs :D

**Comment by danielbreves** [comment]
cc @chucknado

File: README.md [hunk 1]
```diff
@@ -62,6 +63,36 @@ karma start
...
 
+## Deploying
+
+To check that your app will pass the server-side validation check, run
+
+```
+zat validate --path=./dist
+```
+
+If validation is successful, you can upload the app into your Zendesk account by running
```
Comment (chucknado) [inline README.md]: Zendesk account -> Zendesk Support account?

Comment (danielbreves) [inline README.md]: In this case I actually think Zendesk is correct. All apps are uploaded through the Zendesk account, which shared between products.

**Pull request merged**